deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -61,6 +61,13 @@ pub struct MessageObject {
 
     // summary - use/create another function if you need it
     subject: String,
+
+    /// True if the message was correctly encrypted&signed, false otherwise.
+    /// Historically, UIs showed a small padlock on the message then.
+    ///
+    /// Today, the UIs should instead show a small email-icon on the message
```

Comment:
I don't think email icon is the best choice (even if it's already chosen for email-address contacts). Better to show smth encryption-related like "🔓". Moreover, why do we need to show it on messages if we're not going to mix encrypted and unencrypted messages in the same chat anymore? Are there still some cases when a signed+encrypted message may be assigned to a chat with email contacts?

Reply:
yeah, let's stay with letter and move on :) 

- using the same symbol is less to explain. 
-  it feels wrong to have strong warning-signs all over the place, esp, as the user cannot fix that. there is nothing wrong with the ✉️-messages, this is also what the user knows from other clients
- faq needs adaption, sure